compiler/opt.js

```diff
@@ -377,18 +409,6 @@ export default (funcs, globals, pages, tags, exceptions) => {
           continue;
         }
 
-        if (false && i === wasm.length - 1 && inst[0] === Opcodes.return) {
```

Comment:
Why remove this?

Reply:
Woops, I need to find the VSCode extension that's doing that, it removed "dead code" on save.

Reply:
Ah np, just wondered if it was intentional heh